test/vectors/issuance/2.issue-credential_offer.json

```diff
@@ -0,0 +1,198 @@
+                                 ],
+                                 "schema":{
+                                    "type":"string"
+                                 },
+                                 "fallback":"Not disclosed",
+                                 "label":"Gender"
+                              },
+                              {
+                                 "path":[
+                                    "$.credentialSubject.birthDate"
+                                 ],
+                                 "schema":{
+                                    "type":"string",
+                                    "format":"date"
+                                 },
+                                 "label":"Date of Birth"
+                              },
+                              {
+                                 "path":[
+                                    "$.credentialSubject.birthCountry"
+                                 ],
+                                 "schema":{
+                                    "type":"string"
+                                 },
+                                 "label":"Country of Birth"
+                              },
+                              {
+                                 "path":[
+                                    "$.credentialSubject.residentSince"
+                                 ],
+                                 "schema":{
+                                    "type":"string",
+                                    "format":"date"
+                                 },
+                                 "label":"Resident Since"
+                              }
+                           ]
+                        },
+                        "styles":{
+                           "thumbnail":{
+                              "uri":"https://file-server.trustbloc.local:12096/images/credential--uscis-icon.svg",
+                              "alt":"Citizenship & Immigration Services"
+                           },
+                           "hero":{
+                              "uri":"https://example.com/trust.png",
+                              "alt":"Service we trust"
+                           },
+                           "background":{
+                              "color":"#2b5283"
+                           },
+                           "text":{
+                              "color":"#fff"
+                           }
+                        }
+                     }
+                  ],
+                  "id":"dcc75a16-19f5-4273-84ce-4da69ee2b7fe",
+                  "version":"0.1.0",
+                  "issuer":{
+                     "id":"did:example:123?linked-domains=3",
+                     "name":"Government of Example Immigration",
+                     "styles":{
+                        
+                     }
+                  }
+               }
+            }
+         },
+         "format":"dif/credential-manifest/manifest@v1.0",
```

Comment:
format is not a URI, should it be?

Reply:
That's a valid question, but current spec defines it as `dif/credential-manifest/manifest@v1.0`. 

https://identity.foundation/waci-didcomm/#attachments